search/include/pcl/search/organized.h

```diff
@@ -133,7 +133,7 @@ namespace pcl
           input_ = cloud;
           indices_ = indices;
 
-          if (indices_.get () != NULL && indices_->size () != 0)
+          if (indices_.get () != nullptr && indices_->size () != 0)
```

Comment:
Here we don't need to use `get()`, can write simply `if (indices_ && !indices_->empty())`.